src/app/components/BibPage/ElectronicResources.jsx

```diff
@@ -1,22 +1,49 @@
-import { Card, CardContent, CardHeading } from '@nypl/design-system-react-components'
+import { Card, CardContent, CardHeading, Button, Icon } from '@nypl/design-system-react-components'
 import generateERLinksList from '../../utils/electronicResources'
-import React from 'react';
+import React, { useRef, useState } from 'react';
 
 /**
  * ElectronicResources renders a list of electronic resources links, sans aeon links
  * @param {array} electronicResources - an array of electronic resources, passed in as a prop from the BibPage component
+ * @param {boolean} isTestMode - a boolean value only used while running unit tests on this component
  */
-const ElectronicResources = ({ electronicResources }) => {
+
+
+const ElectronicResources = ({ electronicResources, isTestMode = false }) => {
+  const [showMore, setShowMore] = useState(true)
+  const [electronicResourcesToDisplay, setElectronicResourcesToDisplay] = useState(electronicResources);
+  const scrollToRef = useRef();
+
   if (!electronicResources || !electronicResources.length) {
     return null;
   }
-  const resources = generateERLinksList(electronicResources)
-  return (<Card isBordered padding="16px">
+
+  const allResources = generateERLinksList(electronicResources)
+  const threeResources = generateERLinksList(electronicResources.slice(0, 3))
+  const more = `all ${electronicResources.length}`
+  const less = 'fewer'
+
+  const onClick = () => {
+    if (!isTestMode) scrollToRef.current.scrollIntoView({ behavior: 'smooth' })
+    let prevShowMore
+    setShowMore((prev) => {
+      prevShowMore = prev
+      return !prev
+    })
+    setElectronicResourcesToDisplay(prevShowMore ? electronicResources : electronicResources.slice(0, 3))
```

Comment:
This update is good but now you need to use `electronicResourcesToDisplay` instead of `electronicResources`. So you can remove `allResources` and `threeResources` and instead create only one variable that will be rendered.